twilio/stv-twilio-service-test.json

```diff
@@ -1,6 +1,6 @@
 {
-    "account_sid": "xxxx",
-    "auth_token": "xxxx",
-    "notify_service_sid": "xxxx",
-    "phone_number": "xxxx"
-}
\ No newline at end of file
+    "account_sid": "ACcefc75688508570ea7dbe8da2eeeef40",
```

Comment:
Are there any security risks from storing account details in plaintext? Wondering if there's a better way to store this

Reply:
@dzx11 story keys on the server is ok, storing them in the codebase and on GitHub is not. haven't had time to look into the canonical solution for this in a public repo, so treating all keys as ephemeral/test for now. we have an issue tracking here https://github.com/un7c0rn/stopthevirus/issues/53.